### PR TITLE
Add next part of adding kernel support for LLDB

### PR DESCRIPTION
**Pull request by patryk4815**
Changes:
- make gdb only commands available to lldb
- fix lldb issues with pointers 
- improve `klookup` now it search like grep

### CODE REVIEW

File: pwndbg/aglib/symbol.py [hunk 1]
```diff
@@ -106,6 +106,9 @@ def resolve_addr(addr: int) -> str | None:
     Resolution is performed in the following order:
     - Global scope symbols.
     """
+    if addr < 0:
+        return None
```
Comment (disconnect3d) [inline pwndbg/aglib/symbol.py]: Do we need this? Just assert maybe or remove this check?

Comment (patryk4815) [reply pwndbg/aglib/symbol.py]: I can remove this from this PR. Recently someone is sending here negative addresses :/

File: pwndbg/commands/klookup.py [hunk 2]
```diff
@@ -12,21 +12,35 @@
 parser.add_argument("symbol", type=str, help="Address or symbol name to lookup")
 
 
+def parse_to_addr(v: str) -> int:
+    if v.startswith("0x"):
+        return int(v[2:], 16)
+    try:
+        return int(v, 16)
```
Comment (disconnect3d) [inline pwndbg/commands/klookup.py]: We don't need the first if:
```py
In [1]: int('0x1234', 16)
Out[1]: 4660

In [2]: int('1234', 16)
Out[2]: 4660
```

Comment (disconnect3d) [reply pwndbg/commands/klookup.py]: This also means that `klookup <some int in base 10>` will be interpreter as base 16, is that ok?

```py
In [3]: 0xffff1234112
Out[3]: 17591936696594

In [4]: int('17591936696594', 16)
Out[4]: 6571889286276500
```

File: pwndbg/aglib/symbol.py (hunk 1 above)
Comment (patryk4815) [reply pwndbg/aglib/symbol.py]: deleted from this PR

File: pwndbg/commands/klookup.py (hunk 2 above)
Comment (patryk4815) [reply pwndbg/commands/klookup.py]: changed to int(x, 0)

**Pull request merged**